### PR TITLE
Bugfix/bill checking 2024 05 08

### PR DESCRIPTION
**Pull request by priyankatechneai**
resolved bill checking defects

### CODE REVIEW

File: src/screens/BillChecking/BillCheckingTransactions/BillCheckingTransaction.jsx [hunk 1]
```diff
@@ -367,7 +367,7 @@ function BillCheckingTransaction() {
                     row.is_assign_to != 1) ||
                   (row.level != parseInt(row.total_level) &&
                     row.is_approver == 1)) &&
-                row.is_active == 1 && (
+                row.is_active == 1 || row["Is cancelled"]==0 && (
```
Comment (AmitSolnke) [inline src/screens/BillChecking/BillCheckingTransactions/BillCheckingTransaction.jsx:370]: use strict equal to

File: src/screens/BillChecking/AuthorityMapping/AuthorityMapping.jsx [hunk 2]
```diff
@@ -228,6 +227,13 @@ const AuthorityMapping = () => {
       sortable: true,
     },
   ];
+  const [statusData, setstatusData] = useState();
```
Comment (AmitSolnke) [inline src/screens/BillChecking/AuthorityMapping/AuthorityMapping.jsx:230]: where useState are suppose to get declared?

File: src/screens/BillChecking/BillCheckingTransactions/BillCheckingTransaction.jsx (hunk 1 above)
Comment (priyankatechneai) [reply src/screens/BillChecking/BillCheckingTransactions/BillCheckingTransaction.jsx:370]: resolved

File: src/screens/BillChecking/AuthorityMapping/AuthorityMapping.jsx (hunk 2 above)
Comment (priyankatechneai) [reply src/screens/BillChecking/AuthorityMapping/AuthorityMapping.jsx:230]: resolved

**Pull request merged**